docs/cql/ddl.rst

```diff
@@ -961,12 +961,20 @@ A table can be truncated using the ``TRUNCATE`` statement:
 .. code-block::
    
    truncate_statement: TRUNCATE [ TABLE ] `table_name`
+                     : [ USING TIMEOUT `timeout` ]
```

Comment:
Should specify the version that this was added in.

Reply:
The version of what - Scylla? Because we don't have official CQL versioning (see https://github.com/scylladb/scylladb/pull/11320).
I thought we were aiming for versioned documents, so if you look at the documentation of Scylla 5.0 you'll see only the features that Scylla 5.0 supports.

Reply:
It's still good to know when something was added, so you don't get surprised if you run it on an older version and it stops working.

Reply:
Maybe, but then again - almost every feature we added at some point. Do we keep a version comment on every feature forever? Do we only only do this for features introduced in the last year (or whatever), and go back once in a while to remove archaic version information, or what?

Reply:
Why not? e.g. cppreference keeps a version tag for everything. If it's unobtrusive, it isn't a problem.